druid/src/app.rs

```diff
@@ -57,6 +57,7 @@ pub struct WindowConfig {
     pub(crate) min_size: Option<Size>,
     pub(crate) position: Option<Point>,
     pub(crate) resizable: Option<bool>,
+    pub(crate) transparent: Option<bool>,
```

Comment:
I am not really sure why resizable is an option either, but to me it would make sense for this to not be a option. its either transparent or not. (defaulting to false of course)

Reply:
Yeah I agree, however I think the reason it and the others are Option, is probably because it's a builder? In builder pattern I think it's usual to have Option so that to not force the defaults in builder.

Reply:
I cant resolve this for some reason, you'd think it makes sense for the reviewer to be able to mark their own comments are resolved right...
Could you mark it as resolved it for me?